activestorage/lib/active_storage/attached/model.rb

```diff
@@ -32,7 +32,17 @@ module Attached::Model
       #
       # If the +:dependent+ option isn't set, the attachment will be purged
       # (i.e. destroyed) whenever the record is destroyed.
-      def has_one_attached(name, dependent: :purge_later)
+      #
+      # If you need the attachment to use a service, which differs from the globally configured one
+      # you should pass +:service+ option, for instance:
```

Comment:
`If you need the attachment to use a service which differs from the globally configured one, pass the +:service+ option. For instance:`